www/src/components/shell/TerminalSidebar.js

```diff
@@ -422,4 +439,121 @@ function Step3() {
   )
 }
 
+/* ---
+  CHEATSHEET
+--- */
+
+const SECTION_COMMANDS = 'SECTION_COMMANDS'
+const SECTION_DEBUGGING = 'SECTION_DEBUGGING'
+const SECTION_WORKSPACE = 'SECTION_WORKSPACE'
+
+function Cheatsheet() {
+  const [activeSection, setActiveSection] = useState(null)
+  const previousActiveSection = usePrevious(activeSection) || null
+  const displayedSection = activeSection || previousActiveSection
+
+  return (
+    <Div
+      overflow="hidden"
+      maxWidth="100%"
+    >
+      <Flex
+        position="relative"
+        align="flex-start"
+        left={activeSection === null ? 0 : -sidebarWidth}
+        transition="left 333ms ease"
+      >
+        <Div
+          width={sidebarWidth}
+          flexShrink={0}
+        >
+          <CheatsheetItem
+            title="Commands"
+            onClick={() => setActiveSection(SECTION_COMMANDS)}
+          />
+          <CheatsheetItem
+            title="Debugging"
+            onClick={() => setActiveSection(SECTION_DEBUGGING)}
+          />
+          <CheatsheetItem
+            title="Workspace"
+            onClick={() => setActiveSection(SECTION_WORKSPACE)}
+          />
+        </Div>
+        <Div
+          width={sidebarWidth}
+          flexShrink={0}
+          paddingVertical="medium"
+        >
+          <Button
+            small
+            tertiary
+            onClick={() => setActiveSection(null)}
+            marginLeft="medium"
+          >
+            Back to cheatsheet
+          </Button>
+          {displayedSection === SECTION_COMMANDS && (
+            <CheatsheetCommands />
+          )}
+          {displayedSection === SECTION_DEBUGGING && (
+            <CheatsheetDebugging />
+          )}
+          {displayedSection === SECTION_WORKSPACE && (
+            <CheatsheetWorkspace />
+          )}
+        </Div>
+      </Flex>
+    </Div>
+  )
+}
+
+function CheatsheetItem({ children, title, ...props }) {
+  return (
+    <Flex
+      align="center"
+      justify="space-between"
+      paddingHorizontal="large"
+      paddingVertical="medium"
+      borderBottom="1px solid border"
+      cursor="pointer"
+      _hover={{ backgroundColor: 'fill-one-hover' }}
+      {...props}
+    >
+      <P body1>
+        {title}
+      </P>
+      <CollapseIcon
+        transform="rotate(180deg)"
+        color="text-xlight"
+        size={8}
+      />
+    </Flex>
+  )
+}
+
+function CheatsheetCommands() {
+  return (
+    <>
+      CheatsheetCommands
+    </>
+  )
+}
+
+function CheatsheetDebugging() {
+  return (
+    <>
+      CheatsheetDebugging
+    </>
+  )
+}
+
+function CheatsheetWorkspace() {
+  return (
+    <>
+      CheatsheetWorkspace
+    </>
```

Comment:
I need the data for these

Reply:
I'm not actually sure which part of the mocks this refers to.  You should be able to get all the needed data from the query for the cloud shell instance itself